### ISSUE TITLE
mask-image doesnt work in chrome latest

### ISSUE THREAD

**Issue by erichonkanen** [postcss/autoprefixer#881]
Im getting invalid property for the following in chrome:

`mask-image: radial-gradient(circle at 86% 86%, transparent 8px, black 8px);`

![image](https://user-images.githubusercontent.com/230757/29745815-56f20a32-8a7a-11e7-9458-40c343db36f9.png)

**Comment by ai** [comment]
What CSS output you expect?

**Comment by erichonkanen** [comment]
this works in chrome `-webkit-radial-gradient(85% 85%, transparent 7px, black 8px)`

**Comment by ai** [comment]
@erichonkanen can you also try `-webkit-mask-image: -webkit-radial-gradient(85% 85%, circle, transparent 7px, black 8px)`

**Comment by erichonkanen** [comment]
@ai hey, yes that works for chrome but not firefox... this is what im doing currently:

```
    -webkit-mask-image: -webkit-radial-gradient(85% 85%, transparent 7px, black 8px);
    -moz-mask-image: -moz-radial-gradient(circle at 85% 85%, transparent 7px, black 8px);
    mask-image: radial-gradient(circle at 85% 85%, transparent 7px, black 8px);
```

**Comment by ai** [comment]
Yeap seems like radial gradient direction converter doesn’t work correctly (miss `,`).

Use this code until I will make fix:

```css
    -webkit-mask-image: -webkit-radial-gradient(85% 85%, circle, transparent 7px, black 8px);
    mask-image: radial-gradient(circle at 85% 85%, transparent 7px, black 8px);
```

I try to look for issue in this weekend.

**Comment by erichonkanen** [comment]
ok thanks

**Comment by ai** [comment]
Done 46a2d41

**Comment by ai** [comment]
Done 46a2d41

**Comment by ai** [comment]
Released in 7.1.4
